### PR TITLE
Use ActivityResults container object in templates

### PR DESCRIPTION
**Pull request by nickstenning**
This is a small refactoring which changes how the search results are passed into templates. This will make it easier to extract the rendering of the search results into independent partial templates or panels.

### CODE REVIEW

File: h/views/activity.py [hunk 1]
```diff
@@ -75,14 +75,12 @@ def user_link(userid):
...
-            'aggregations': result.aggregations,
+            'search_results': results,
             'groups_suggestions': groups_suggestions,
-            'page': paginate(self.request, result.total, page_size=page_size),
+            'page': paginate(self.request, results.total, page_size=page_size),
             'pretty_link': pretty_link,
             'q': self.request.params.get('q', ''),
             'tag_link': tag_link,
-            'timeframes': result.timeframes,
-            'total': result.total,
```
Comment (chdorner) [inline h/views/activity.py]: I can't seem to add comments to lines that are not in the initial set of the diff.

Line 408 in the search template still uses the old `total`, that needs to change to `search_results.total` as well

**Comment by chdorner** [comment]
Pushed the fix. Waiting for Travis and will then merge.

**Comment by robertknight** [comment]
Rebased. Just waiting on Travis.

File: h/views/activity.py (hunk 1 above)
Comment (nickstenning) [inline h/views/activity.py]: Thanks for spotting this one!

**Comment by chdorner** [comment]
Oh, I must have forgotten to merge it yesterday. Thanks for taking over @robertknight

**Pull request merged**